// @rc-component/dialog/dist/common-async.js
((typeof globalThis !== 'undefined' ? globalThis : self)["makoChunk_@rc-component/dialog"] = (typeof globalThis !== 'undefined' ? globalThis : self)["makoChunk_@rc-component/dialog"] || []).push([
        ['common'],
{ "docs/changelog.md?type=text": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "texts", {
    enumerable: true,
    get: function() {
        return texts;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
__mako_require__("docs/changelog.md?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const texts = [
    {
        "value": "2021-01-07",
        "paraId": 0,
        "tocIndex": 1
    },
    {
        "value": "improve: ScrollLocker call related. ",
        "paraId": 1,
        "tocIndex": 1
    },
    {
        "value": "#227",
        "paraId": 1,
        "tocIndex": 1
    },
    {
        "value": "2021-01-04",
        "paraId": 2,
        "tocIndex": 2
    },
    {
        "value": "refactor: use ScrollLocker. ",
        "paraId": 3,
        "tocIndex": 2
    },
    {
        "value": "#220",
        "paraId": 3,
        "tocIndex": 2
    },
    {
        "value": "2020-12-19",
        "paraId": 4,
        "tocIndex": 3
    },
    {
        "value": "fix: trigger close only when click wrap itself. ",
        "paraId": 5,
        "tocIndex": 3
    },
    {
        "value": "#218",
        "paraId": 5,
        "tocIndex": 3
    },
    {
        "value": "2020-12-07",
        "paraId": 6,
        "tocIndex": 4
    },
    {
        "value": "fix: Dialog should keep refresh when forceRender provided. ",
        "paraId": 7,
        "tocIndex": 4
    },
    {
        "value": "#217",
        "paraId": 7,
        "tocIndex": 4
    },
    {
        "value": "2020-12-03",
        "paraId": 8,
        "tocIndex": 5
    },
    {
        "value": "fix: dialog dont close when mouseDown in content and mouseUp in wrapper. ",
        "paraId": 9,
        "tocIndex": 5
    },
    {
        "value": "#210",
        "paraId": 9,
        "tocIndex": 5
    },
    {
        "value": "chore: Should not re-render when dialog is invisible. ",
        "paraId": 10,
        "tocIndex": 5
    },
    {
        "value": "#212",
        "paraId": 10,
        "tocIndex": 5
    },
    {
        "value": "2020-10-21",
        "paraId": 11,
        "tocIndex": 6
    },
    {
        "value": "chore: support react 17. ",
        "paraId": 12,
        "tocIndex": 6
    },
    {
        "value": "#207",
        "paraId": 12,
        "tocIndex": 6
    },
    {
        "value": "2020-10-14",
        "paraId": 13,
        "tocIndex": 7
    },
    {
        "value": "fix: Dialog should not auto destroy. ",
        "paraId": 14,
        "tocIndex": 7
    },
    {
        "value": "#206",
        "paraId": 14,
        "tocIndex": 7
    },
    {
        "value": "2020-10-11",
        "paraId": 15,
        "tocIndex": 8
    },
    {
        "value": "fix: Portal event bubble. ",
        "paraId": 16,
        "tocIndex": 8
    },
    {
        "value": "#204",
        "paraId": 16,
        "tocIndex": 8
    },
    {
        "value": "2020-09-29",
        "paraId": 17,
        "tocIndex": 9
    },
    {
        "value": "refactor: Use ",
        "paraId": 18,
        "tocIndex": 9
    },
    {
        "value": "rc-motion",
        "paraId": 18,
        "tocIndex": 9
    },
    {
        "value": ". ",
        "paraId": 18,
        "tocIndex": 9
    },
    {
        "value": "#203",
        "paraId": 18,
        "tocIndex": 9
    },
    {
        "value": "fix: prevent scroll behavior when focus trigger. ",
        "paraId": 19,
        "tocIndex": 10
    },
    {
        "value": "ant-design/ant-design#26582",
        "paraId": 19,
        "tocIndex": 10
    },
    {
        "value": "fix: page scroll position will jump after dialog is closed. ",
        "paraId": 20,
        "tocIndex": 11
    },
    {
        "value": "#202",
        "paraId": 20,
        "tocIndex": 11
    },
    {
        "value": "fix: remove typing from package.json. ",
        "paraId": 21,
        "tocIndex": 12
    },
    {
        "value": "#201",
        "paraId": 21,
        "tocIndex": 12
    },
    {
        "value": "fix: add displayName. ",
        "paraId": 22,
        "tocIndex": 13
    },
    {
        "value": "#200",
        "paraId": 22,
        "tocIndex": 13
    },
    {
        "value": "fate: add ",
        "paraId": 23,
        "tocIndex": 14
    },
    {
        "value": "modalRender",
        "paraId": 23,
        "tocIndex": 14
    },
    {
        "value": ". ",
        "paraId": 23,
        "tocIndex": 14
    },
    {
        "value": "#195",
        "paraId": 23,
        "tocIndex": 14
    },
    {
        "value": "use ",
        "paraId": 24,
        "tocIndex": 15
    },
    {
        "value": "father",
        "paraId": 24,
        "tocIndex": 15
    },
    {
        "value": ". ",
        "paraId": 24,
        "tocIndex": 15
    },
    {
        "value": "#197",
        "paraId": 24,
        "tocIndex": 15
    },
    {
        "value": "Fix dialog component will only show mask, if initialize a Dialog component with both forceRender and visible are true. ",
        "paraId": 25,
        "tocIndex": 16
    },
    {
        "value": "#194",
        "paraId": 25,
        "tocIndex": 16
    },
    {
        "value": "remove babel runtime.",
        "paraId": 26,
        "tocIndex": 17
    },
    {
        "value": "up ",
        "paraId": 26,
        "tocIndex": 17
    },
    {
        "value": "rc-drawer",
        "paraId": 26,
        "tocIndex": 17
    },
    {
        "value": " to ",
        "paraId": 26,
        "tocIndex": 17
    },
    {
        "value": "4.1.0",
        "paraId": 26,
        "tocIndex": 17
    },
    {
        "value": ".",
        "paraId": 26,
        "tocIndex": 17
    },
    {
        "value": "upgrade ",
        "paraId": 27,
        "tocIndex": 18
    },
    {
        "value": "rc-util",
        "paraId": 27,
        "tocIndex": 18
    },
    {
        "value": " to ",
        "paraId": 27,
        "tocIndex": 18
    },
    {
        "value": "5.0.0",
        "paraId": 27,
        "tocIndex": 18
    },
    {
        "value": ".",
        "paraId": 27,
        "tocIndex": 18
    },
    {
        "value": "upgrade ",
        "paraId": 28,
        "tocIndex": 19
    },
    {
        "value": "rc-animate",
        "paraId": 28,
        "tocIndex": 19
    },
    {
        "value": " to ",
        "paraId": 28,
        "tocIndex": 19
    },
    {
        "value": "3.0.0",
        "paraId": 28,
        "tocIndex": 19
    },
    {
        "value": ".",
        "paraId": 28,
        "tocIndex": 19
    },
    {
        "value": "Update accessibility.",
        "paraId": 29,
        "tocIndex": 20
    },
    {
        "value": "Support ",
        "paraId": 30,
        "tocIndex": 21
    },
    {
        "value": "forceRender",
        "paraId": 30,
        "tocIndex": 21
    },
    {
        "value": " for dialog.",
        "paraId": 30,
        "tocIndex": 21
    },
    {
        "value": "Add closeIcon. ",
        "paraId": 31,
        "tocIndex": 22
    },
    {
        "value": "#89",
        "paraId": 32,
        "tocIndex": 22
    },
    {
        "value": " ",
        "paraId": 31,
        "tocIndex": 22
    },
    {
        "value": "@HeskeyBaozi ",
        "paraId": 31,
        "tocIndex": 22
    },
    {
        "value": "Add destroyOnClose. ",
        "paraId": 33,
        "tocIndex": 23
    },
    {
        "value": "#72",
        "paraId": 33,
        "tocIndex": 23
    },
    {
        "value": " ",
        "paraId": 33,
        "tocIndex": 23
    },
    {
        "value": "@Rohanhacker",
        "paraId": 33,
        "tocIndex": 23
    },
    {
        "value": "Remove ReactNative support, please use ",
        "paraId": 34,
        "tocIndex": 24
    },
    {
        "value": "https://github.com/react-component/m-dialog",
        "paraId": 34,
        "tocIndex": 24
    },
    {
        "value": " instead.",
        "paraId": 34,
        "tocIndex": 24
    },
    {
        "value": "Support React 16.",
        "paraId": 34,
        "tocIndex": 24
    },
    {
        "value": "Notable change: Close animation won't trigger when dialog unmounting after React 16, see ",
        "paraId": 35,
        "tocIndex": 24
    },
    {
        "value": "facebook/react#10826",
        "paraId": 35,
        "tocIndex": 24
    },
    {
        "value": "fixed: RN modal support landscape orientation, ",
        "paraId": 36,
        "tocIndex": 25
    },
    {
        "value": "https://github.com/react-component/dialog/pull/64",
        "paraId": 36,
        "tocIndex": 25
    },
    {
        "value": "remove rc-dialog/lib/Modal's entry prop, add animationType prop",
        "paraId": 37,
        "tocIndex": 26
    },
    {
        "value": "add rc-dialog/lib/Modal to support react-native",
        "paraId": 38,
        "tocIndex": 27
    },
    {
        "value": "use getContainerRenderMixin from 'rc-util'",
        "paraId": 39,
        "tocIndex": 28
    },
    {
        "value": "new html structure and class",
        "paraId": 40,
        "tocIndex": 29
    },
    {
        "value": "disable window scroll when show",
        "paraId": 40,
        "tocIndex": 29
    },
    {
        "value": "add maskClosable",
        "paraId": 41,
        "tocIndex": 30
    },
    {
        "value": "separate close and header",
        "paraId": 42,
        "tocIndex": 31
    },
    {
        "value": "only support react 0.14",
        "paraId": 43,
        "tocIndex": 32
    },
    {
        "value": "refactor to clean api. remove onShow onBeforeClose",
        "paraId": 44,
        "tocIndex": 33
    },
    {
        "value": "use rc-animate & rc-align",
        "paraId": 45,
        "tocIndex": 34
    },
    {
        "value": "support esc to close",
        "paraId": 46,
        "tocIndex": 35
    },
    {
        "value": "add renderToBody props",
        "paraId": 47,
        "tocIndex": 36
    },
    {
        "value": "make dialog render to body and use ",
        "paraId": 48,
        "tocIndex": 37
    },
    {
        "value": "dom-align",
        "paraId": 48,
        "tocIndex": 37
    },
    {
        "value": " to align",
        "paraId": 48,
        "tocIndex": 37
    },
    {
        "value": "support es6 and react 0.13",
        "paraId": 49,
        "tocIndex": 38
    },
    {
        "value": "new",
        "paraId": 50,
        "tocIndex": 39
    },
    {
        "value": " ",
        "paraId": 50,
        "tocIndex": 39
    },
    {
        "value": "#3",
        "paraId": 50,
        "tocIndex": 39
    },
    {
        "value": " support closable requestClose onBeforeClose",
        "paraId": 50,
        "tocIndex": 39
    }
];
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/demo/ant-design.md?type=text": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "texts", {
    enumerable: true,
    get: function() {
        return texts;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
__mako_require__("docs/demo/ant-design.md?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const texts = [];
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/demo/bootstrap.md?type=text": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "texts", {
    enumerable: true,
    get: function() {
        return texts;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
__mako_require__("docs/demo/bootstrap.md?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const texts = [];
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/demo/draggable.md?type=text": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "texts", {
    enumerable: true,
    get: function() {
        return texts;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
__mako_require__("docs/demo/draggable.md?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const texts = [];
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/demo/pure.md?type=text": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "texts", {
    enumerable: true,
    get: function() {
        return texts;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
__mako_require__("docs/demo/pure.md?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const texts = [];
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/examples/ant-design.tsx?watch=parent": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/examples/bootstrap.tsx?watch=parent": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/examples/draggable.tsx?watch=parent": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/examples/multiple-Portal.tsx?watch=parent": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/examples/pure.tsx?watch=parent": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/index.md?type=text": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "texts", {
    enumerable: true,
    get: function() {
        return texts;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
__mako_require__("docs/index.md?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const texts = [
    {
        "value": "react dialog component.",
        "paraId": 0,
        "tocIndex": 0
    },
    {
        "value": " ",
        "paraId": 1,
        "tocIndex": 0
    },
    {
        "value": " ",
        "paraId": 1,
        "tocIndex": 0
    },
    {
        "value": " ",
        "paraId": 1,
        "tocIndex": 0
    },
    {
        "value": " ",
        "paraId": 1,
        "tocIndex": 0
    },
    {
        "value": " ",
        "paraId": 1,
        "tocIndex": 0
    },
    {
        "value": "http://localhost:8007/examples/",
        "paraId": 2,
        "tocIndex": 2
    },
    {
        "value": "online example: ",
        "paraId": 3,
        "tocIndex": 2
    },
    {
        "value": "https://dialog.react-component.vercel.app/",
        "paraId": 3,
        "tocIndex": 2
    },
    {
        "value": "var Dialog = require('@rc-component/dialog');\n\nReactDOM.render(\n  <Dialog title={title} onClose={callback1} visible>\n      <p>first dialog</p>\n  </Dialog>\n), document.getElementById('t1'));\n\n// use dialog\n",
        "paraId": 4,
        "tocIndex": 4
    },
    {
        "value": "Name",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Type",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Default",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Description",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Version",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "prefixCls",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "String",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "rc-dialog",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "The dialog dom node's prefixCls",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "className",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "String",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "additional className for dialog",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "classNames",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "{ header?: string; body?: string; footer?: string; mask?: string; content?: string; wrapper?: string; }",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "pass className to target area",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "styles",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "{ header?: CSSProperties; body?: CSSProperties; footer?: CSSProperties; mask?: CSSProperties; content?: CSSProperties; wrapper?: CSSProperties; }",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "pass styles to target area",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "style",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Object",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "{}",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Root style for dialog element.Such as width, height",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "zIndex",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Number",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "visible",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Boolean",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "false",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "current dialog's visible status",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "animation",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "String",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "part of dialog animation css class name",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "maskAnimation",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "String",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "part of dialog's mask animation css class name",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "transitionName",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "String",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "dialog animation css class name",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "maskTransitionName",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "String",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "mask animation css class name",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "title",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "String|React.Element",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Title of the dialog",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "footer",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "React.Element",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "footer of the dialog",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "closable",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Boolean | ({ closeIcon?: React.ReactNode; disabled?: boolean, afterClose:function } & React.AriaAttributes)",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "true",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "whether show close button",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "mask",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Boolean",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "true",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "whether show mask",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "maskClosable",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Boolean",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "true",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "whether click mask to close",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "keyboard",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Boolean",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "true",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "whether support press esc to close",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "mousePosition",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "{x:number,y:number}",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "set pageX and pageY of current mouse(it will cause transform origin to be set).",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "onClose",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "function()",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "called when click close button or mask",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "afterClose",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "function()",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "called when close animation end",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "getContainer",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "function(): HTMLElement",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "to determine where Dialog will be mounted",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "destroyOnHidden",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Boolean",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "false",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "to unmount child compenents on onClose",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "closeIcon",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "ReactNode",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "specific the close icon.",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "forceRender",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Boolean",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "false",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Create dialog dom node before dialog first show",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "focusTriggerAfterClose",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Boolean",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "true",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "focus trigger element when dialog closed",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "modalRender",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "(node: ReactNode) => ReactNode",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Custom modal content render",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "8.3.0",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "npm install\nnpm start\n",
        "paraId": 6,
        "tocIndex": 7
    },
    {
        "value": "npm test\nnpm run chrome-test\n",
        "paraId": 7,
        "tocIndex": 8
    },
    {
        "value": "npm run coverage\n",
        "paraId": 8,
        "tocIndex": 9
    },
    {
        "value": "open coverage/ dir",
        "paraId": 9,
        "tocIndex": 9
    },
    {
        "value": "@rc-component/dialog is released under the MIT license.",
        "paraId": 10,
        "tocIndex": 10
    },
    {
        "value": "\n  ",
        "paraId": 11
    }
];
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"src/Dialog/Content/MemoChildren.tsx": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react/index.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _default = /*#__PURE__*/ _c1 = _react.memo(_c = ({ children })=>children, (_, { shouldUpdate })=>!shouldUpdate);
var _c;
var _c1;
$RefreshReg$(_c, "%default%$React.memo");
$RefreshReg$(_c1, "%default%");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"src/Dialog/Content/Panel.tsx": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/react/jsx-dev-runtime.js");
var _clsx = __mako_require__("node_modules/clsx/dist/clsx.mjs");
var _ref = __mako_require__("node_modules/@rc-component/util/lib/ref.js");
var _focus = __mako_require__("node_modules/@rc-component/util/lib/Dom/focus.js");
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react/index.js"));
var _context = __mako_require__("src/context.ts");
var _MemoChildren = /*#__PURE__*/ _interop_require_default._(__mako_require__("src/Dialog/Content/MemoChildren.tsx"));
var _pickAttrs = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/@rc-component/util/lib/pickAttrs.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _s = $RefreshSig$();
const Panel = /*#__PURE__*/ _s(_react.default.forwardRef(_c = _s((props, ref)=>{
    _s();
    const { prefixCls, className, style, title, ariaId, footer, closable, closeIcon, onClose, children, bodyStyle, bodyProps, modalRender, onMouseDown, onMouseUp, holderRef, visible, forceRender, width, height, classNames: modalClassNames, styles: modalStyles } = props;
    // ================================= Refs =================================
    const { panel: panelRef } = _react.default.useContext(_context.RefContext);
    const internalRef = (0, _react.useRef)(null);
    const mergedRef = (0, _ref.useComposeRef)(holderRef, panelRef, internalRef);
    (0, _focus.useLockFocus)(visible, ()=>internalRef.current);
    _react.default.useImperativeHandle(ref, ()=>({
            focus: ()=>{
                var _internalRef_current;
                (_internalRef_current = internalRef.current) === null || _internalRef_current === void 0 || _internalRef_current.focus({
                    preventScroll: true
                });
            }
        }));
    // ================================ Style =================================
    const contentStyle = {};
    if (width !== undefined) contentStyle.width = width;
    if (height !== undefined) contentStyle.height = height;
    // ================================ Render ================================
    const footerNode = footer ? /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
        className: (0, _clsx.clsx)(`${prefixCls}-footer`, modalClassNames === null || modalClassNames === void 0 ? void 0 : modalClassNames.footer),
        style: {
            ...modalStyles === null || modalStyles === void 0 ? void 0 : modalStyles.footer
        },
        children: footer
    }, void 0, false, {
        fileName: "src/Dialog/Content/Panel.tsx",
        lineNumber: 72,
        columnNumber: 5
    }, this) : null;
    const headerNode = title ? /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
        className: (0, _clsx.clsx)(`${prefixCls}-header`, modalClassNames === null || modalClassNames === void 0 ? void 0 : modalClassNames.header),
        style: {
            ...modalStyles === null || modalStyles === void 0 ? void 0 : modalStyles.header
        },
        children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
            className: (0, _clsx.clsx)(`${prefixCls}-title`, modalClassNames === null || modalClassNames === void 0 ? void 0 : modalClassNames.title),
            id: ariaId,
            style: {
                ...modalStyles === null || modalStyles === void 0 ? void 0 : modalStyles.title
            },
            children: title
        }, void 0, false, {
            fileName: "src/Dialog/Content/Panel.tsx",
            lineNumber: 85,
            columnNumber: 7
        }, this)
    }, void 0, false, {
        fileName: "src/Dialog/Content/Panel.tsx",
        lineNumber: 81,
        columnNumber: 5
    }, this) : null;
    const closableObj = (0, _react.useMemo)(()=>{
        if (typeof closable === 'object' && closable !== null) return closable;
        if (closable) return {
            closeIcon: closeIcon ?? /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                className: `${prefixCls}-close-x`
            }, void 0, false, {
                fileName: "src/Dialog/Content/Panel.tsx",
                lineNumber: 100,
                columnNumber: 40
            }, this)
        };
        return {};
    }, [
        closable,
        closeIcon,
        prefixCls
    ]);
    const ariaProps = (0, _pickAttrs.default)(closableObj, true);
    const closeBtnIsDisabled = typeof closable === 'object' && closable.disabled;
    const closerNode = closable ? /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
        type: "button",
        onClick: onClose,
        "aria-label": "Close",
        ...ariaProps,
        className: `${prefixCls}-close`,
        disabled: closeBtnIsDisabled,
        children: closableObj.closeIcon
    }, void 0, false, {
        fileName: "src/Dialog/Content/Panel.tsx",
        lineNumber: 109,
        columnNumber: 5
    }, this) : null;
    const content = /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
        className: (0, _clsx.clsx)(`${prefixCls}-container`, modalClassNames === null || modalClassNames === void 0 ? void 0 : modalClassNames.container),
        style: modalStyles === null || modalStyles === void 0 ? void 0 : modalStyles.container,
        children: [
            closerNode,
            headerNode,
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
                className: (0, _clsx.clsx)(`${prefixCls}-body`, modalClassNames === null || modalClassNames === void 0 ? void 0 : modalClassNames.body),
                style: {
                    ...bodyStyle,
                    ...modalStyles === null || modalStyles === void 0 ? void 0 : modalStyles.body
                },
                ...bodyProps,
                children: children
            }, void 0, false, {
                fileName: "src/Dialog/Content/Panel.tsx",
                lineNumber: 128,
                columnNumber: 7
            }, this),
            footerNode
        ]
    }, void 0, true, {
        fileName: "src/Dialog/Content/Panel.tsx",
        lineNumber: 122,
        columnNumber: 5
    }, this);
    return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
        role: "dialog",
        "aria-labelledby": title ? ariaId : null,
        "aria-modal": "true",
        ref: mergedRef,
        style: {
            ...style,
            ...contentStyle
        },
        className: (0, _clsx.clsx)(prefixCls, className),
        onMouseDown: onMouseDown,
        onMouseUp: onMouseUp,
        tabIndex: -1,
        children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_MemoChildren.default, {
            shouldUpdate: visible || forceRender,
            children: modalRender ? modalRender(content) : content
        }, void 0, false, {
            fileName: "src/Dialog/Content/Panel.tsx",
            lineNumber: 152,
            columnNumber: 7
        }, this)
    }, "dialog-element", false, {
        fileName: "src/Dialog/Content/Panel.tsx",
        lineNumber: 140,
        columnNumber: 5
    }, this);
}, "bU247zidmyUZOs2UoXntPMSnEJE=", false, function() {
    return [
        _ref.useComposeRef,
        _focus.useLockFocus
    ];
})), "bU247zidmyUZOs2UoXntPMSnEJE=", false, function() {
    return [
        _ref.useComposeRef,
        _focus.useLockFocus
    ];
});
_c1 = Panel;
Panel.displayName = 'Panel';
var _default = Panel;
var _c;
var _c1;
$RefreshReg$(_c, "Panel$React.forwardRef");
$RefreshReg$(_c1, "Panel");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"src/Dialog/Content/index.tsx": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/react/jsx-dev-runtime.js");
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react/index.js"));
var _clsx = __mako_require__("node_modules/clsx/dist/clsx.mjs");
var _motion = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/@rc-component/motion/es/index.js"));
var _util = __mako_require__("src/util.ts");
var _Panel = /*#__PURE__*/ _interop_require_default._(__mako_require__("src/Dialog/Content/Panel.tsx"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _s = $RefreshSig$();
const Content = /*#__PURE__*/ _s(_react.forwardRef(_c = _s((props, ref)=>{
    _s();
    const { prefixCls, title, style, className, visible, forceRender, destroyOnHidden, motionName, ariaId, onVisibleChanged, mousePosition } = props;
    const dialogRef = (0, _react.useRef)(null);
    const panelRef = (0, _react.useRef)(null);
    // ============================== Refs ==============================
    _react.useImperativeHandle(ref, ()=>({
            ...panelRef.current,
            inMotion: dialogRef.current.inMotion,
            enableMotion: dialogRef.current.enableMotion
        }));
    // ============================= Style ==============================
    const [transformOrigin, setTransformOrigin] = _react.useState();
    const contentStyle = {};
    if (transformOrigin) contentStyle.transformOrigin = transformOrigin;
    function onPrepare() {
        const elementOffset = (0, _util.offset)(dialogRef.current.nativeElement);
        setTransformOrigin(mousePosition && (mousePosition.x || mousePosition.y) ? `${mousePosition.x - elementOffset.left}px ${mousePosition.y - elementOffset.top}px` : '');
    }
    // ============================= Render =============================
    return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_motion.default, {
        visible: visible,
        onVisibleChanged: onVisibleChanged,
        onAppearPrepare: onPrepare,
        onEnterPrepare: onPrepare,
        forceRender: forceRender,
        motionName: motionName,
        removeOnLeave: destroyOnHidden,
        ref: dialogRef,
        children: ({ className: motionClassName, style: motionStyle }, motionRef)=>/*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_Panel.default, {
                ...props,
                ref: panelRef,
                title: title,
                ariaId: ariaId,
                prefixCls: prefixCls,
                holderRef: motionRef,
                style: {
                    ...motionStyle,
                    ...style,
                    ...contentStyle
                },
                className: (0, _clsx.clsx)(className, motionClassName)
            }, void 0, false, {
                fileName: "src/Dialog/Content/index.tsx",
                lineNumber: 77,
                columnNumber: 9
            }, this)
    }, void 0, false, {
        fileName: "src/Dialog/Content/index.tsx",
        lineNumber: 66,
        columnNumber: 5
    }, this);
}, "B8iJBl3QRco45KFPnPJ2XBRLfiA=")), "B8iJBl3QRco45KFPnPJ2XBRLfiA=");
_c1 = Content;
Content.displayName = 'Content';
var _default = Content;
var _c;
var _c1;
$RefreshReg$(_c, "Content$React.forwardRef");
$RefreshReg$(_c1, "Content");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"src/Dialog/Mask.tsx": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/react/jsx-dev-runtime.js");
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react/index.js"));
var _clsx = __mako_require__("node_modules/clsx/dist/clsx.mjs");
var _motion = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/@rc-component/motion/es/index.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const Mask = (props)=>{
    const { prefixCls, style, visible, maskProps, motionName, className } = props;
    return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_motion.default, {
        visible: visible,
        motionName: motionName,
        leavedClassName: `${prefixCls}-mask-hidden`,
        children: ({ className: motionClassName, style: motionStyle }, ref)=>/*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
                ref: ref,
                style: {
                    ...motionStyle,
                    ...style
                },
                className: (0, _clsx.clsx)(`${prefixCls}-mask`, motionClassName, className),
                ...maskProps
            }, void 0, false, {
                fileName: "src/Dialog/Mask.tsx",
                lineNumber: 24,
                columnNumber: 9
            }, this)
    }, "mask", false, {
        fileName: "src/Dialog/Mask.tsx",
        lineNumber: 17,
        columnNumber: 5
    }, this);
};
_c = Mask;
var _default = Mask;
var _c;
$RefreshReg$(_c, "Mask");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"src/Dialog/index.tsx": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/react/jsx-dev-runtime.js");
var _clsx = __mako_require__("node_modules/clsx/dist/clsx.mjs");
var _contains = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/@rc-component/util/lib/Dom/contains.js"));
var _useId = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/@rc-component/util/lib/hooks/useId.js"));
var _KeyCode = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/@rc-component/util/lib/KeyCode.js"));
var _pickAttrs = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/@rc-component/util/lib/pickAttrs.js"));
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react/index.js"));
var _util = __mako_require__("src/util.ts");
var _Content = /*#__PURE__*/ _interop_require_default._(__mako_require__("src/Dialog/Content/index.tsx"));
var _Mask = /*#__PURE__*/ _interop_require_default._(__mako_require__("src/Dialog/Mask.tsx"));
var _warning = __mako_require__("node_modules/@rc-component/util/lib/warning.js");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _s = $RefreshSig$();
const Dialog = (props)=>{
    _s();
    const { prefixCls = 'rc-dialog', zIndex, visible = false, keyboard = true, focusTriggerAfterClose = true, // scrollLocker,
    // Wrapper
    wrapStyle, wrapClassName, wrapProps, onClose, afterOpenChange, afterClose, // Dialog
    transitionName, animation, closable = true, // Mask
    mask = true, maskTransitionName, maskAnimation, maskClosable = true, maskStyle, maskProps, rootClassName, rootStyle, classNames: modalClassNames, styles: modalStyles } = props;
    [
        'wrapStyle',
        'bodyStyle',
        'maskStyle'
    ].forEach((prop)=>{
        (0, _warning.warning)(!(prop in props), `${prop} is deprecated, please use styles instead.`);
    });
    if ('wrapClassName' in props) (0, _warning.warning)(false, `wrapClassName is deprecated, please use classNames instead.`);
    const lastOutSideActiveElementRef = (0, _react.useRef)(null);
    const wrapperRef = (0, _react.useRef)(null);
    const contentRef = (0, _react.useRef)(null);
    const [animatedVisible, setAnimatedVisible] = _react.useState(visible);
    // ========================== Init ==========================
    const ariaId = (0, _useId.default)();
    function saveLastOutSideActiveElementRef() {
        if (!(0, _contains.default)(wrapperRef.current, document.activeElement)) lastOutSideActiveElementRef.current = document.activeElement;
    }
    function focusDialogContent() {
        if (!(0, _contains.default)(wrapperRef.current, document.activeElement)) {
            var _contentRef_current;
            (_contentRef_current = contentRef.current) === null || _contentRef_current === void 0 || _contentRef_current.focus();
        }
    }
    // ========================= Events =========================
    // Close action will trigger by:
    //   1. When hide motion end
    //   2. Controlled `open` to `false` immediately after set to `true` which will not trigger motion
    function doClose() {
        // Clean up scroll bar & focus back
        setAnimatedVisible(false);
        if (mask && lastOutSideActiveElementRef.current && focusTriggerAfterClose) {
            try {
                lastOutSideActiveElementRef.current.focus({
                    preventScroll: true
                });
            } catch (e) {
            // Do nothing
            }
            lastOutSideActiveElementRef.current = null;
        }
        // Trigger afterClose only when change visible from true to false
        if (animatedVisible) afterClose === null || afterClose === void 0 || afterClose();
    }
    function onDialogVisibleChanged(newVisible) {
        // Try to focus
        if (newVisible) focusDialogContent();
        else doClose();
        afterOpenChange === null || afterOpenChange === void 0 || afterOpenChange(newVisible);
    }
    function onInternalClose(e) {
        onClose === null || onClose === void 0 || onClose(e);
    }
    // >>> Content
    const contentClickRef = (0, _react.useRef)(false);
    const contentTimeoutRef = (0, _react.useRef)(null);
    // We need record content click incase content popup out of dialog
    const onContentMouseDown = ()=>{
        clearTimeout(contentTimeoutRef.current);
        contentClickRef.current = true;
    };
    const onContentMouseUp = ()=>{
        contentTimeoutRef.current = setTimeout(()=>{
            contentClickRef.current = false;
        });
    };
    // >>> Wrapper
    // Close only when element not on dialog
    let onWrapperClick = null;
    if (maskClosable) onWrapperClick = (e)=>{
        if (contentClickRef.current) contentClickRef.current = false;
        else if (wrapperRef.current === e.target) onInternalClose(e);
    };
    function onWrapperKeyDown(e) {
        if (keyboard && e.keyCode === _KeyCode.default.ESC) {
            e.stopPropagation();
            onInternalClose(e);
            return;
        }
    }
    // ========================= Effect =========================
    (0, _react.useEffect)(()=>{
        if (visible) {
            setAnimatedVisible(true);
            saveLastOutSideActiveElementRef();
        } else if (animatedVisible && contentRef.current.enableMotion() && !contentRef.current.inMotion()) doClose();
    }, [
        visible
    ]);
    // Remove direct should also check the scroll bar update
    (0, _react.useEffect)(()=>()=>{
            clearTimeout(contentTimeoutRef.current);
        }, []);
    const mergedStyle = {
        zIndex,
        ...wrapStyle,
        ...modalStyles === null || modalStyles === void 0 ? void 0 : modalStyles.wrapper,
        display: !animatedVisible ? 'none' : null
    };
    // ========================= Render =========================
    return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
        className: (0, _clsx.clsx)(`${prefixCls}-root`, rootClassName),
        style: rootStyle,
        ...(0, _pickAttrs.default)(props, {
            data: true
        }),
        children: [
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_Mask.default, {
                prefixCls: prefixCls,
                visible: mask && visible,
                motionName: (0, _util.getMotionName)(prefixCls, maskTransitionName, maskAnimation),
                style: {
                    zIndex,
                    ...maskStyle,
                    ...modalStyles === null || modalStyles === void 0 ? void 0 : modalStyles.mask
                },
                maskProps: maskProps,
                className: modalClassNames === null || modalClassNames === void 0 ? void 0 : modalClassNames.mask
            }, void 0, false, {
                fileName: "src/Dialog/index.tsx",
                lineNumber: 188,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
                onKeyDown: onWrapperKeyDown,
                className: (0, _clsx.clsx)(`${prefixCls}-wrap`, wrapClassName, modalClassNames === null || modalClassNames === void 0 ? void 0 : modalClassNames.wrapper),
                ref: wrapperRef,
                onClick: onWrapperClick,
                style: mergedStyle,
                ...wrapProps,
                children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_Content.default, {
                    ...props,
                    onMouseDown: onContentMouseDown,
                    onMouseUp: onContentMouseUp,
                    ref: contentRef,
                    closable: closable,
                    ariaId: ariaId,
                    prefixCls: prefixCls,
                    visible: visible && animatedVisible,
                    onClose: onInternalClose,
                    onVisibleChanged: onDialogVisibleChanged,
                    motionName: (0, _util.getMotionName)(prefixCls, transitionName, animation)
                }, void 0, false, {
                    fileName: "src/Dialog/index.tsx",
                    lineNumber: 204,
                    columnNumber: 9
                }, this)
            }, void 0, false, {
                fileName: "src/Dialog/index.tsx",
                lineNumber: 196,
                columnNumber: 7
            }, this)
        ]
    }, void 0, true, {
        fileName: "src/Dialog/index.tsx",
        lineNumber: 183,
        columnNumber: 5
    }, this);
};
_s(Dialog, "jpgAGSop6biDTxPFPAMT85ShMu0=", false, function() {
    return [
        _useId.default
    ];
});
_c = Dialog;
var _default = Dialog;
var _c;
$RefreshReg$(_c, "Dialog");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"src/DialogWrap.tsx": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/react/jsx-dev-runtime.js");
var _portal = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/@rc-component/portal/es/index.js"));
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react/index.js"));
var _context = __mako_require__("src/context.ts");
var _Dialog = /*#__PURE__*/ _interop_require_default._(__mako_require__("src/Dialog/index.tsx"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _s = $RefreshSig$();
// fix issue #10656
/*
 * getContainer remarks
 * Custom container should not be return, because in the Portal component, it will remove the
 * return container element here, if the custom container is the only child of it's component,
 * like issue #10656, It will has a conflict with removeChild method in react-dom.
 * So here should add a child (div element) to custom container.
 * */ const DialogWrap = (props)=>{
    _s();
    const { visible, getContainer, forceRender, destroyOnHidden = false, afterClose, closable, panelRef } = props;
    const [animatedVisible, setAnimatedVisible] = _react.useState(visible);
    const refContext = _react.useMemo(()=>({
            panel: panelRef
        }), [
        panelRef
    ]);
    _react.useEffect(()=>{
        if (visible) setAnimatedVisible(true);
    }, [
        visible
    ]);
    // Destroy on close will remove wrapped div
    if (!forceRender && destroyOnHidden && !animatedVisible) return null;
    return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_context.RefContext.Provider, {
        value: refContext,
        children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_portal.default, {
            open: visible || forceRender || animatedVisible,
            autoDestroy: false,
            getContainer: getContainer,
            autoLock: visible || animatedVisible,
            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_Dialog.default, {
                ...props,
                destroyOnHidden: destroyOnHidden,
                afterClose: ()=>{
                    const closableObj = closable && typeof closable === 'object' ? closable : {};
                    const { afterClose: closableAfterClose } = closableObj || {};
                    closableAfterClose === null || closableAfterClose === void 0 || closableAfterClose();
                    afterClose === null || afterClose === void 0 || afterClose();
                    setAnimatedVisible(false);
                }
            }, void 0, false, {
                fileName: "src/DialogWrap.tsx",
                lineNumber: 49,
                columnNumber: 9
            }, this)
        }, void 0, false, {
            fileName: "src/DialogWrap.tsx",
            lineNumber: 43,
            columnNumber: 7
        }, this)
    }, void 0, false, {
        fileName: "src/DialogWrap.tsx",
        lineNumber: 42,
        columnNumber: 5
    }, this);
};
_s(DialogWrap, "N2Fqxpoym3ZwLNsWl2JCF4u+lvs=");
_c = DialogWrap;
DialogWrap.displayName = 'Dialog';
var _default = DialogWrap;
var _c;
$RefreshReg$(_c, "DialogWrap");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"src/context.ts": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "RefContext", {
    enumerable: true,
    get: function() {
        return RefContext;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react/index.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const RefContext = /*#__PURE__*/ _react.createContext({});
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"src/index.ts": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
function _export(target, all) {
    for(var name in all)Object.defineProperty(target, name, {
        enumerable: true,
        get: all[name]
    });
}
__mako_require__.e(exports, {
    Panel: function() {
        return _Panel.default;
    },
    default: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var _DialogWrap = /*#__PURE__*/ _interop_require_default._(__mako_require__("src/DialogWrap.tsx"));
var _Panel = /*#__PURE__*/ _interop_require_default._(__mako_require__("src/Dialog/Content/Panel.tsx"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _default = _DialogWrap.default;
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"src/util.ts": function (module, exports, __mako_require__){
// =============================== Motion ===============================
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
function _export(target, all) {
    for(var name in all)Object.defineProperty(target, name, {
        enumerable: true,
        get: all[name]
    });
}
__mako_require__.e(exports, {
    getMotionName: function() {
        return getMotionName;
    },
    offset: function() {
        return offset;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
function getMotionName(prefixCls, transitionName, animationName) {
    let motionName = transitionName;
    if (!motionName && animationName) motionName = `${prefixCls}-${animationName}`;
    return motionName;
}
// =============================== Offset ===============================
function getScroll(w, top) {
    let ret = w[`page${top ? 'Y' : 'X'}Offset`];
    const method = `scroll${top ? 'Top' : 'Left'}`;
    if (typeof ret !== 'number') {
        const d = w.document;
        ret = d.documentElement[method];
        if (typeof ret !== 'number') ret = d.body[method];
    }
    return ret;
}
function offset(el) {
    const rect = el.getBoundingClientRect();
    const pos = {
        left: rect.left,
        top: rect.top
    };
    const doc = el.ownerDocument;
    const w = doc.defaultView || doc.parentWindow;
    pos.left += getScroll(w);
    pos.top += getScroll(w, true);
    return pos;
}
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
 }]);
