// @rc-component/dialog/dist/demos-async.js
((typeof globalThis !== 'undefined' ? globalThis : self)["makoChunk_@rc-component/dialog"] = (typeof globalThis !== 'undefined' ? globalThis : self)["makoChunk_@rc-component/dialog"] || []).push([
        ['demos'],
{ "docs/examples/ant-design.tsx?techStack=react": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/react/jsx-dev-runtime.js");
__mako_require__("docs/examples/ant-design.tsx?watch=parent");
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react/index.js"));
var _select = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/@rc-component/select/es/index.js"));
"";
var _dialog = /*#__PURE__*/ _interop_require_default._(__mako_require__("src/index.ts"));
"";
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _s = $RefreshSig$();
const clearPath = "M793 242H366v-74c0-6.7-7.7-10.4-12.9-6.3l-142 112c-4.1 3.2-4.1 9.4 0 12.6l142 112c5.2 4.1 12.9 0.4 12.9-6.3v-74h415v470H175c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h618c35.3 0 64-28.7 64-64V306c0-35.3-28.7-64-64-64z";
const getSvg = (path, props = {}, align = false)=>/*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("i", {
        ...props,
        children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("svg", {
            viewBox: "0 0 1024 1024",
            width: "1em",
            height: "1em",
            fill: "currentColor",
            style: align ? {
                verticalAlign: '-0.125em '
            } : {},
            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("path", {
                d: path
            }, void 0, false, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 24,
                columnNumber: 7
            }, this)
        }, void 0, false, {
            fileName: "docs/examples/ant-design.tsx?techStack=react",
            lineNumber: 17,
            columnNumber: 5
        }, this)
    }, void 0, false, {
        fileName: "docs/examples/ant-design.tsx?techStack=react",
        lineNumber: 16,
        columnNumber: 3
    }, this);
const MyControl = ()=>{
    _s();
    const [visible1, setVisible1] = _react.useState(true);
    const [visible2, setVisible2] = _react.useState(false);
    const [visible3, setVisible3] = _react.useState(false);
    const [width, setWidth] = _react.useState(600);
    const [destroyOnHidden, setDestroyOnHidden] = _react.useState(false);
    const [center, setCenter] = _react.useState(false);
    const [mousePosition, setMousePosition] = _react.useState({
        x: null,
        y: null
    });
    const [useIcon, setUseIcon] = _react.useState(false);
    const [forceRender, setForceRender] = _react.useState(false);
    const onClick = (e)=>{
        setMousePosition({
            x: e.pageX,
            y: e.pageY
        });
        setVisible1(true);
    };
    const onClose = ()=>{
        setVisible1(false);
    };
    const onClose2 = ()=>{
        setVisible2(false);
    };
    const onClose3 = ()=>{
        setVisible3(false);
    };
    const closeAll = ()=>{
        setVisible1(false);
        setVisible2(false);
        setVisible3(false);
    };
    const onDestroyOnHiddenChange = (e)=>{
        setDestroyOnHidden(e.target.checked);
    };
    const onForceRenderChange = (e)=>{
        setForceRender(e.target.checked);
    };
    const changeWidth = ()=>{
        setWidth(width === 600 ? 800 : 600);
    };
    const centerEvent = (e)=>{
        setCenter(e.target.checked);
    };
    const toggleCloseIcon = ()=>{
        setUseIcon(!useIcon);
    };
    const style = {
        width
    };
    let wrapClassName = '';
    if (center) wrapClassName = 'center';
    const dialog = /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_dialog.default, {
        visible: visible1,
        wrapClassName: wrapClassName,
        animation: "zoom",
        maskAnimation: "fade",
        onClose: onClose,
        style: style,
        title: "dialog1",
        mousePosition: mousePosition,
        destroyOnHidden: destroyOnHidden,
        closeIcon: useIcon ? getSvg(clearPath, {}, true) : undefined,
        forceRender: forceRender,
        focusTriggerAfterClose: false,
        children: [
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("input", {
                autoFocus: true
            }, void 0, false, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 105,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                children: "basic modal"
            }, void 0, false, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 106,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                type: "button",
                onClick: ()=>{
                    setVisible1(false);
                    setVisible2(true);
                },
                children: "打开第二个并关闭当前的"
            }, void 0, false, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 107,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                type: "button",
                onClick: ()=>{
                    setVisible2(true);
                },
                children: "打开第二个"
            }, void 0, false, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 116,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                type: "button",
                onClick: changeWidth,
                children: "change width"
            }, void 0, false, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 124,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                type: "button",
                onClick: toggleCloseIcon,
                children: [
                    "use custom icon, is using icon: ",
                    useIcon && 'true' || 'false',
                    "."
                ]
            }, void 0, true, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 127,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
                style: {
                    height: 200
                },
                children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_select.default, {
                    popupStyle: {
                        zIndex: 9999999
                    },
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_select.default.Option, {
                        value: "light",
                        children: "Light"
                    }, void 0, false, {
                        fileName: "docs/examples/ant-design.tsx?techStack=react",
                        lineNumber: 132,
                        columnNumber: 11
                    }, this)
                }, void 0, false, {
                    fileName: "docs/examples/ant-design.tsx?techStack=react",
                    lineNumber: 131,
                    columnNumber: 9
                }, this)
            }, void 0, false, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 130,
                columnNumber: 7
            }, this)
        ]
    }, void 0, true, {
        fileName: "docs/examples/ant-design.tsx?techStack=react",
        lineNumber: 91,
        columnNumber: 5
    }, this);
    const dialog2 = /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_dialog.default, {
        visible: visible2,
        title: "dialog2",
        onClose: onClose2,
        children: [
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("input", {
                autoFocus: true
            }, void 0, false, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 140,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                children: "basic modal"
            }, void 0, false, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 141,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                type: "button",
                onClick: ()=>{
                    setVisible3(true);
                },
                children: "打开第三个"
            }, void 0, false, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 142,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                type: "button",
                onClick: ()=>{
                    setVisible2(false);
                },
                children: "关闭当前"
            }, void 0, false, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 150,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                type: "button",
                onClick: closeAll,
                children: "关闭所有"
            }, void 0, false, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 158,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                type: "button",
                onClick: changeWidth,
                children: "change width"
            }, void 0, false, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 161,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                type: "button",
                onClick: toggleCloseIcon,
                children: [
                    "use custom icon, is using icon: ",
                    useIcon && 'true' || 'false',
                    "."
                ]
            }, void 0, true, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 164,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
                style: {
                    height: 200
                }
            }, void 0, false, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 167,
                columnNumber: 7
            }, this)
        ]
    }, void 0, true, {
        fileName: "docs/examples/ant-design.tsx?techStack=react",
        lineNumber: 139,
        columnNumber: 5
    }, this);
    const dialog3 = /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_dialog.default, {
        forceRender: true,
        title: "dialog3",
        visible: visible3,
        onClose: onClose3,
        children: [
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                children: "initialized with forceRender and visbile true"
            }, void 0, false, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 173,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                type: "button",
                onClick: ()=>{
                    setVisible3(false);
                },
                children: "关闭当前"
            }, void 0, false, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 174,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                type: "button",
                onClick: closeAll,
                children: "关闭所有"
            }, void 0, false, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 182,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                type: "button",
                onClick: changeWidth,
                children: "change width"
            }, void 0, false, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 185,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                type: "button",
                onClick: toggleCloseIcon,
                children: [
                    "use custom icon, is using icon: ",
                    useIcon && 'true' || 'false',
                    "."
                ]
            }, void 0, true, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 188,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
                style: {
                    height: 200
                }
            }, void 0, false, {
                fileName: "docs/examples/ant-design.tsx?techStack=react",
                lineNumber: 191,
                columnNumber: 7
            }, this)
        ]
    }, void 0, true, {
        fileName: "docs/examples/ant-design.tsx?techStack=react",
        lineNumber: 172,
        columnNumber: 5
    }, this);
    return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_react.StrictMode, {
        children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
            style: {
                width: '90%',
                margin: '0 auto',
                height: '150vh'
            },
            children: [
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("style", {
                    children: `
          .center {
            display: flex;
            align-items: center;
            justify-content: center;
          }
          `
                }, void 0, false, {
                    fileName: "docs/examples/ant-design.tsx?techStack=react",
                    lineNumber: 198,
                    columnNumber: 9
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                            type: "button",
                            className: "btn btn-primary",
                            onClick: onClick,
                            children: "show dialog"
                        }, void 0, false, {
                            fileName: "docs/examples/ant-design.tsx?techStack=react",
                            lineNumber: 208,
                            columnNumber: 11
                        }, this),
                        " ",
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("label", {
                            children: [
                                "destroy on hidden:",
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("input", {
                                    type: "checkbox",
                                    checked: destroyOnHidden,
                                    onChange: onDestroyOnHiddenChange
                                }, void 0, false, {
                                    fileName: "docs/examples/ant-design.tsx?techStack=react",
                                    lineNumber: 214,
                                    columnNumber: 13
                                }, this)
                            ]
                        }, void 0, true, {
                            fileName: "docs/examples/ant-design.tsx?techStack=react",
                            lineNumber: 212,
                            columnNumber: 11
                        }, this),
                        " ",
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("label", {
                            children: [
                                "center",
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("input", {
                                    type: "checkbox",
                                    checked: center,
                                    onChange: centerEvent
                                }, void 0, false, {
                                    fileName: "docs/examples/ant-design.tsx?techStack=react",
                                    lineNumber: 219,
                                    columnNumber: 13
                                }, this)
                            ]
                        }, void 0, true, {
                            fileName: "docs/examples/ant-design.tsx?techStack=react",
                            lineNumber: 217,
                            columnNumber: 11
                        }, this),
                        " ",
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("label", {
                            children: [
                                "force render",
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("input", {
                                    type: "checkbox",
                                    checked: forceRender,
                                    onChange: onForceRenderChange
                                }, void 0, false, {
                                    fileName: "docs/examples/ant-design.tsx?techStack=react",
                                    lineNumber: 224,
                                    columnNumber: 13
                                }, this)
                            ]
                        }, void 0, true, {
                            fileName: "docs/examples/ant-design.tsx?techStack=react",
                            lineNumber: 222,
                            columnNumber: 11
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("input", {
                            placeholder: "Useless Input",
                            onClick: onClick
                        }, void 0, false, {
                            fileName: "docs/examples/ant-design.tsx?techStack=react",
                            lineNumber: 226,
                            columnNumber: 11
                        }, this)
                    ]
                }, void 0, true, {
                    fileName: "docs/examples/ant-design.tsx?techStack=react",
                    lineNumber: 207,
                    columnNumber: 9
                }, this),
                dialog,
                dialog2,
                dialog3
            ]
        }, void 0, true, {
            fileName: "docs/examples/ant-design.tsx?techStack=react",
            lineNumber: 197,
            columnNumber: 7
        }, this)
    }, void 0, false, {
        fileName: "docs/examples/ant-design.tsx?techStack=react",
        lineNumber: 196,
        columnNumber: 5
    }, this);
};
_s(MyControl, "1JhJ+SabN+SWeiQ+w4YKu6iAhLE=");
_c = MyControl;
var _default = MyControl;
var _c;
$RefreshReg$(_c, "MyControl");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/examples/bootstrap.tsx?techStack=react": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/react/jsx-dev-runtime.js");
__mako_require__("docs/examples/bootstrap.tsx?watch=parent");
var _react = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/react/index.js"));
var _dialog = /*#__PURE__*/ _interop_require_default._(__mako_require__("src/index.ts"));
"";
"";
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _s = $RefreshSig$();
// Check for memo update should work
const InnerRender = ()=>{
    console.log('Updated...', Date.now());
    return null;
};
_c = InnerRender;
const MyControl = ()=>{
    _s();
    const [visible, setVisible] = _react.default.useState(false);
    const [destroyOnHidden, setDestroyOnHidden] = _react.default.useState(false);
    const onClick = ()=>{
        setVisible(true);
    };
    const onClose = ()=>{
        setVisible(false);
    };
    const onDestroyOnHiddenChange = (e)=>{
        setDestroyOnHidden(e.target.checked);
    };
    const dialog = /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_dialog.default, {
        visible: visible,
        destroyOnHidden: destroyOnHidden,
        animation: "slide-fade",
        maskAnimation: "fade",
        onClose: onClose,
        style: {
            width: 600
        },
        title: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
            children: "第二个弹框"
        }, void 0, false, {
            fileName: "docs/examples/bootstrap.tsx?techStack=react",
            lineNumber: 36,
            columnNumber: 14
        }, void 0),
        footer: [
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                type: "button",
                className: "btn btn-default",
                onClick: onClose,
                children: "Close"
            }, "close", false, {
                fileName: "docs/examples/bootstrap.tsx?techStack=react",
                lineNumber: 38,
                columnNumber: 9
            }, void 0),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                type: "button",
                className: "btn btn-primary",
                onClick: onClose,
                children: "Save changes"
            }, "save", false, {
                fileName: "docs/examples/bootstrap.tsx?techStack=react",
                lineNumber: 41,
                columnNumber: 9
            }, void 0)
        ],
        children: [
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(InnerRender, {}, void 0, false, {
                fileName: "docs/examples/bootstrap.tsx?techStack=react",
                lineNumber: 46,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h4", {
                children: "Text in a modal"
            }, void 0, false, {
                fileName: "docs/examples/bootstrap.tsx?techStack=react",
                lineNumber: 47,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                children: "Duis mollis, est non commodo luctus, nisi erat porttitor ligula."
            }, void 0, false, {
                fileName: "docs/examples/bootstrap.tsx?techStack=react",
                lineNumber: 48,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("hr", {}, void 0, false, {
                fileName: "docs/examples/bootstrap.tsx?techStack=react",
                lineNumber: 49,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h4", {
                children: "Overflowing text to show scroll behavior"
            }, void 0, false, {
                fileName: "docs/examples/bootstrap.tsx?techStack=react",
                lineNumber: 50,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                children: "Cras mattis consectetur purus sit amet fermentum. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Morbi leo risus, porta ac consectetur ac, vestibulum at eros."
            }, void 0, false, {
                fileName: "docs/examples/bootstrap.tsx?techStack=react",
                lineNumber: 51,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                children: [
                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                        type: "button",
                        className: "btn btn-primary",
                        children: "Primary"
                    }, void 0, false, {
                        fileName: "docs/examples/bootstrap.tsx?techStack=react",
                        lineNumber: 56,
                        columnNumber: 9
                    }, this),
                    ' ',
                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                        type: "button",
                        className: "btn btn-secondary",
                        children: "Secondary"
                    }, void 0, false, {
                        fileName: "docs/examples/bootstrap.tsx?techStack=react",
                        lineNumber: 59,
                        columnNumber: 9
                    }, this),
                    ' ',
                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                        type: "button",
                        className: "btn btn-success",
                        children: "Success"
                    }, void 0, false, {
                        fileName: "docs/examples/bootstrap.tsx?techStack=react",
                        lineNumber: 62,
                        columnNumber: 9
                    }, this),
                    ' ',
                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                        type: "button",
                        className: "btn btn-danger",
                        children: "Danger"
                    }, void 0, false, {
                        fileName: "docs/examples/bootstrap.tsx?techStack=react",
                        lineNumber: 65,
                        columnNumber: 9
                    }, this),
                    ' ',
                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                        type: "button",
                        className: "btn btn-warning",
                        children: "Warning"
                    }, void 0, false, {
                        fileName: "docs/examples/bootstrap.tsx?techStack=react",
                        lineNumber: 68,
                        columnNumber: 9
                    }, this),
                    ' ',
                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                        type: "button",
                        className: "btn btn-info",
                        children: "Info"
                    }, void 0, false, {
                        fileName: "docs/examples/bootstrap.tsx?techStack=react",
                        lineNumber: 71,
                        columnNumber: 9
                    }, this),
                    ' '
                ]
            }, void 0, true, {
                fileName: "docs/examples/bootstrap.tsx?techStack=react",
                lineNumber: 55,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                children: "Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor."
            }, void 0, false, {
                fileName: "docs/examples/bootstrap.tsx?techStack=react",
                lineNumber: 75,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
                style: {
                    display: ''
                },
                children: [
                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                        children: "Aenean lacinia bibendum nulla sed consectetur. Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Donec sed odio dui. Donec ullamcorper nulla non metus auctor fringilla."
                    }, void 0, false, {
                        fileName: "docs/examples/bootstrap.tsx?techStack=react",
                        lineNumber: 80,
                        columnNumber: 9
                    }, this),
                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                        children: "Cras mattis consectetur purus sit amet fermentum. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Morbi leo risus, porta ac consectetur ac, vestibulum at eros."
                    }, void 0, false, {
                        fileName: "docs/examples/bootstrap.tsx?techStack=react",
                        lineNumber: 85,
                        columnNumber: 9
                    }, this),
                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                        children: "Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor."
                    }, void 0, false, {
                        fileName: "docs/examples/bootstrap.tsx?techStack=react",
                        lineNumber: 89,
                        columnNumber: 9
                    }, this),
                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                        children: "Aenean lacinia bibendum nulla sed consectetur. Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Donec sed odio dui. Donec ullamcorper nulla non metus auctor fringilla."
                    }, void 0, false, {
                        fileName: "docs/examples/bootstrap.tsx?techStack=react",
                        lineNumber: 93,
                        columnNumber: 9
                    }, this),
                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                        children: "Cras mattis consectetur purus sit amet fermentum. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Morbi leo risus, porta ac consectetur ac, vestibulum at eros."
                    }, void 0, false, {
                        fileName: "docs/examples/bootstrap.tsx?techStack=react",
                        lineNumber: 98,
                        columnNumber: 9
                    }, this),
                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                        children: "Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor."
                    }, void 0, false, {
                        fileName: "docs/examples/bootstrap.tsx?techStack=react",
                        lineNumber: 102,
                        columnNumber: 9
                    }, this),
                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                        children: "Aenean lacinia bibendum nulla sed consectetur. Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Donec sed odio dui. Donec ullamcorper nulla non metus auctor fringilla."
                    }, void 0, false, {
                        fileName: "docs/examples/bootstrap.tsx?techStack=react",
                        lineNumber: 106,
                        columnNumber: 9
                    }, this)
                ]
            }, void 0, true, {
                fileName: "docs/examples/bootstrap.tsx?techStack=react",
                lineNumber: 79,
                columnNumber: 7
            }, this)
        ]
    }, void 0, true, {
        fileName: "docs/examples/bootstrap.tsx?techStack=react",
        lineNumber: 29,
        columnNumber: 5
    }, this);
    return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
        style: {
            margin: 20
        },
        children: [
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                children: [
                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                        type: "button",
                        className: "btn btn-primary",
                        onClick: onClick,
                        children: "show dialog"
                    }, void 0, false, {
                        fileName: "docs/examples/bootstrap.tsx?techStack=react",
                        lineNumber: 118,
                        columnNumber: 9
                    }, this),
                    " ",
                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("label", {
                        children: [
                            "destroy on hidden:",
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("input", {
                                type: "checkbox",
                                checked: destroyOnHidden,
                                onChange: onDestroyOnHiddenChange
                            }, void 0, false, {
                                fileName: "docs/examples/bootstrap.tsx?techStack=react",
                                lineNumber: 124,
                                columnNumber: 11
                            }, this)
                        ]
                    }, void 0, true, {
                        fileName: "docs/examples/bootstrap.tsx?techStack=react",
                        lineNumber: 122,
                        columnNumber: 9
                    }, this)
                ]
            }, void 0, true, {
                fileName: "docs/examples/bootstrap.tsx?techStack=react",
                lineNumber: 117,
                columnNumber: 7
            }, this),
            dialog
        ]
    }, void 0, true, {
        fileName: "docs/examples/bootstrap.tsx?techStack=react",
        lineNumber: 116,
        columnNumber: 5
    }, this);
};
_s(MyControl, "Nh1DLC/Dg3sgOQ6X8E9k3j5RyRk=");
_c1 = MyControl;
var _default = MyControl;
var _c;
var _c1;
$RefreshReg$(_c, "InnerRender");
$RefreshReg$(_c1, "MyControl");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/examples/draggable.tsx?techStack=react": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/react/jsx-dev-runtime.js");
__mako_require__("docs/examples/draggable.tsx?watch=parent");
"";
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react/index.js"));
var _reactdraggable = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/react-draggable/build/cjs/cjs.js"));
var _dialog = /*#__PURE__*/ _interop_require_default._(__mako_require__("src/index.ts"));
"";
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _s = $RefreshSig$();
const MyControl = ()=>{
    _s();
    const [visible, setVisible] = _react.useState(false);
    const [disabled, setDisabled] = _react.useState(true);
    const onClick = ()=>{
        setVisible(true);
    };
    const onClose = ()=>{
        setVisible(false);
    };
    return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
        style: {
            margin: 20
        },
        children: [
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                    type: "button",
                    className: "btn btn-primary",
                    onClick: onClick,
                    children: "show dialog"
                }, void 0, false, {
                    fileName: "docs/examples/draggable.tsx?techStack=react",
                    lineNumber: 19,
                    columnNumber: 9
                }, this)
            }, void 0, false, {
                fileName: "docs/examples/draggable.tsx?techStack=react",
                lineNumber: 18,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_dialog.default, {
                visible: visible,
                animation: "slide-fade",
                maskAnimation: "fade",
                onClose: onClose,
                style: {
                    width: 600
                },
                title: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
                    style: {
                        width: '100%',
                        cursor: 'pointer'
                    },
                    onMouseOver: ()=>{
                        if (disabled) setDisabled(false);
                    },
                    onMouseOut: ()=>{
                        setDisabled(true);
                    },
                    // fix eslintjsx-a11y/mouse-events-have-key-events
                    // https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/mouse-events-have-key-events.md
                    onFocus: ()=>{},
                    onBlur: ()=>{},
                    children: "modal"
                }, void 0, false, {
                    fileName: "docs/examples/draggable.tsx?techStack=react",
                    lineNumber: 30,
                    columnNumber: 11
                }, void 0),
                modalRender: (modal)=>/*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_reactdraggable.default, {
                        disabled: disabled,
                        children: modal
                    }, void 0, false, {
                        fileName: "docs/examples/draggable.tsx?techStack=react",
                        lineNumber: 49,
                        columnNumber: 33
                    }, void 0),
                children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
                    style: {
                        height: 200
                    },
                    children: "Day before yesterday I saw a rabbit, and yesterday a deer, and today, you."
                }, void 0, false, {
                    fileName: "docs/examples/draggable.tsx?techStack=react",
                    lineNumber: 51,
                    columnNumber: 9
                }, this)
            }, void 0, false, {
                fileName: "docs/examples/draggable.tsx?techStack=react",
                lineNumber: 23,
                columnNumber: 7
            }, this)
        ]
    }, void 0, true, {
        fileName: "docs/examples/draggable.tsx?techStack=react",
        lineNumber: 17,
        columnNumber: 5
    }, this);
};
_s(MyControl, "QI8aYP7T5NAIm89XPwPafPceRPo=");
_c = MyControl;
var _default = MyControl;
var _c;
$RefreshReg$(_c, "MyControl");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/examples/multiple-Portal.tsx?techStack=react": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/react/jsx-dev-runtime.js");
__mako_require__("docs/examples/multiple-Portal.tsx?watch=parent");
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react/index.js"));
var _drawer = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/@rc-component/drawer/es/index.js"));
"";
var _dialog = /*#__PURE__*/ _interop_require_default._(__mako_require__("src/index.ts"));
"";
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _s = $RefreshSig$();
const Demo = ()=>{
    _s();
    const [showDialog, setShowDialog] = _react.useState(false);
    const [showDrawer, setShowDrawer] = _react.useState(false);
    const onToggleDrawer = ()=>{
        setShowDrawer((value)=>!value);
    };
    const onToggleDialog = ()=>{
        setShowDialog((value)=>!value);
    };
    const dialog = /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_dialog.default, {
        visible: showDialog,
        animation: "zoom",
        maskAnimation: "fade",
        onClose: onToggleDialog,
        forceRender: true,
        title: "basic modal",
        children: [
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                    type: "button",
                    onClick: onToggleDrawer,
                    children: "show drawer"
                }, void 0, false, {
                    fileName: "docs/examples/multiple-Portal.tsx?techStack=react",
                    lineNumber: 29,
                    columnNumber: 9
                }, this)
            }, void 0, false, {
                fileName: "docs/examples/multiple-Portal.tsx?techStack=react",
                lineNumber: 28,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
                style: {
                    height: 200
                }
            }, void 0, false, {
                fileName: "docs/examples/multiple-Portal.tsx?techStack=react",
                lineNumber: 33,
                columnNumber: 7
            }, this)
        ]
    }, void 0, true, {
        fileName: "docs/examples/multiple-Portal.tsx?techStack=react",
        lineNumber: 20,
        columnNumber: 5
    }, this);
    const drawer = /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_drawer.default, {
        open: showDrawer,
        onClose: onToggleDrawer,
        children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
            type: "button",
            onClick: onToggleDrawer,
            children: "close drawer"
        }, void 0, false, {
            fileName: "docs/examples/multiple-Portal.tsx?techStack=react",
            lineNumber: 38,
            columnNumber: 7
        }, this)
    }, void 0, false, {
        fileName: "docs/examples/multiple-Portal.tsx?techStack=react",
        lineNumber: 37,
        columnNumber: 5
    }, this);
    return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
        children: [
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                type: "button",
                onClick: onToggleDialog,
                children: "open dialog"
            }, void 0, false, {
                fileName: "docs/examples/multiple-Portal.tsx?techStack=react",
                lineNumber: 45,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
                type: "button",
                onClick: ()=>{
                    setShowDialog(true);
                    setTimeout(()=>{
                        setShowDialog(false);
                    }, 0);
                },
                children: "quick"
            }, void 0, false, {
                fileName: "docs/examples/multiple-Portal.tsx?techStack=react",
                lineNumber: 48,
                columnNumber: 7
            }, this),
            dialog,
            drawer
        ]
    }, void 0, true, {
        fileName: "docs/examples/multiple-Portal.tsx?techStack=react",
        lineNumber: 44,
        columnNumber: 5
    }, this);
};
_s(Demo, "NXmLaLBTsMoIATQhEKlsH30qgiI=");
_c = Demo;
var _default = Demo;
var _c;
$RefreshReg$(_c, "Demo");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/examples/pure.tsx?techStack=react": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/react/jsx-dev-runtime.js");
__mako_require__("docs/examples/pure.tsx?watch=parent");
var _react = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/react/index.js"));
var _dialog = __mako_require__("src/index.ts");
"";
"";
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const Demo = ()=>/*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_dialog.Panel, {
        prefixCls: "rc-dialog",
        title: "Title",
        closable: true,
        children: "Hello World!"
    }, void 0, false, {
        fileName: "docs/examples/pure.tsx?techStack=react",
        lineNumber: 7,
        columnNumber: 3
    }, this);
_c = Demo;
var _default = Demo;
var _c;
$RefreshReg$(_c, "Demo");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
 }]);
