// @rc-component/dialog/dist/docs_changelog_md-async.js
((typeof globalThis !== 'undefined' ? globalThis : self)["makoChunk_@rc-component/dialog"] = (typeof globalThis !== 'undefined' ? globalThis : self)["makoChunk_@rc-component/dialog"] || []).push([
        ['docs/changelog.md'],
{ "CHANGELOG.md?watch=parent": function (module, exports, __mako_require__){

},
"docs/changelog.md": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/react/jsx-dev-runtime.js");
var _index = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/dumi/theme-default/builtins/API/index.js"));
var _index1 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/dumi/theme-default/builtins/Badge/index.js"));
var _index2 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/dumi/theme-default/builtins/CodeGroup/index.js"));
var _index3 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/dumi/theme-default/builtins/Container/index.js"));
var _index4 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/dumi/theme-default/builtins/Previewer/index.js"));
var _index5 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/dumi/theme-default/builtins/SourceCode/index.js"));
var _index6 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/dumi/theme-default/builtins/Table/index.js"));
var _index7 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/dumi/theme-default/builtins/Tree/index.js"));
var _dumi = __mako_require__(".dumi/tmp/dumi/exports.ts");
__mako_require__("CHANGELOG.md?watch=parent");
var _loading = /*#__PURE__*/ _interop_require_default._(__mako_require__(".dumi/tmp/dumi/theme/loading.tsx"));
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react/index.js"));
var _changelogmdtypetext = __mako_require__("docs/changelog.md?type=text");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
// export named function for fastRefresh
// ref: https://github.com/pmmmwh/react-refresh-webpack-plugin/blob/main/docs/TROUBLESHOOTING.md#edits-always-lead-to-full-reload
function DumiMarkdownContent() {
    return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_dumi.DumiPage, {
        children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_react.Suspense, {
            fallback: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_loading.default, {}, void 0, false, {
                fileName: "docs/changelog.md",
                lineNumber: 25,
                columnNumber: 27
            }, void 0),
            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_jsxdevruntime.Fragment, {
                children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
                    className: "markdown",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h1", {
                            id: "changelog",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#changelog",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 110
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 56
                                }, this),
                                "Changelog"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 37
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "851",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#851",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 228
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 180
                                }, this),
                                "8.5.1"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 167
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                children: _changelogmdtypetext.texts[0].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 284
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 281
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[1].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/react-component/dialog/pull/227",
                                        children: _changelogmdtypetext.texts[2].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 359
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 330
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 326
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "850",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#850",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 520
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 472
                                }, this),
                                "8.5.0"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 459
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                children: _changelogmdtypetext.texts[3].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 576
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 573
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[4].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/react-component/dialog/pull/220",
                                        children: _changelogmdtypetext.texts[5].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 651
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 622
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 618
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "846",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#846",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 812
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 764
                                }, this),
                                "8.4.6"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 751
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                children: _changelogmdtypetext.texts[6].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 868
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 865
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[7].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/react-component/dialog/pull/218",
                                        children: _changelogmdtypetext.texts[8].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 943
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 914
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 910
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "845",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#845",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 1104
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 1056
                                }, this),
                                "8.4.5"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 1043
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                children: _changelogmdtypetext.texts[9].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 1160
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 1157
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[10].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/react-component/dialog/pull/217",
                                        children: _changelogmdtypetext.texts[11].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 1236
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 1206
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 1202
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "844",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#844",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 1398
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 1350
                                }, this),
                                "8.4.4"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 1337
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                children: _changelogmdtypetext.texts[12].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 1454
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 1451
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                                        children: [
                                            _changelogmdtypetext.texts[13].value,
                                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                                href: "https://github.com/react-component/dialog/pull/210",
                                                children: _changelogmdtypetext.texts[14].value
                                            }, void 0, false, {
                                                fileName: "docs/changelog.md",
                                                lineNumber: 26,
                                                columnNumber: 1534
                                            }, this)
                                        ]
                                    }, void 0, true, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 1505
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 1501
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                                        children: [
                                            _changelogmdtypetext.texts[15].value,
                                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                                href: "https://github.com/react-component/dialog/pull/212",
                                                children: _changelogmdtypetext.texts[16].value
                                            }, void 0, false, {
                                                fileName: "docs/changelog.md",
                                                lineNumber: 26,
                                                columnNumber: 1667
                                            }, this)
                                        ]
                                    }, void 0, true, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 1638
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 1634
                                }, this)
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 1497
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "843",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#843",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 1833
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 1785
                                }, this),
                                "8.4.3"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 1772
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                children: _changelogmdtypetext.texts[17].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 1889
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 1886
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[18].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/react-component/dialog/pull/207",
                                        children: _changelogmdtypetext.texts[19].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 1966
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 1936
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 1932
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "842",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#842",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 2128
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 2080
                                }, this),
                                "8.4.2"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 2067
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                children: _changelogmdtypetext.texts[20].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 2184
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 2181
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[21].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/react-component/dialog/pull/206",
                                        children: _changelogmdtypetext.texts[22].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 2261
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 2231
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 2227
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "841",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#841",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 2423
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 2375
                                }, this),
                                "8.4.1"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 2362
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                children: _changelogmdtypetext.texts[23].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 2479
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 2476
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[24].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/react-component/dialog/pull/204",
                                        children: _changelogmdtypetext.texts[25].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 2556
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 2526
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 2522
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "840",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#840",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 2718
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 2670
                                }, this),
                                "8.4.0"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 2657
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                children: _changelogmdtypetext.texts[26].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 2774
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 2771
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[27].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                        children: _changelogmdtypetext.texts[28].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 2851
                                    }, this),
                                    _changelogmdtypetext.texts[29].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/react-component/dialog/pull/203",
                                        children: _changelogmdtypetext.texts[30].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 2916
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 2821
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 2817
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "834-822-812--2020-09-04",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#834-822-812--2020-09-04",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 3118
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 3050
                                }, this),
                                "8.3.4 (8.2.2, 8.1.2) / 2020-09-04"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 3017
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[31].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/ant-design/ant-design/issues/26582",
                                        children: _changelogmdtypetext.texts[32].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 3233
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 3203
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 3199
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "833--2020-09-02",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#833--2020-09-02",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 3422
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 3362
                                }, this),
                                "8.3.3 / 2020-09-02"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 3337
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[33].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/react-component/dialog/pull/202",
                                        children: _changelogmdtypetext.texts[34].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 3522
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 3492
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 3488
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "832--2020-09-02",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#832--2020-09-02",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 3708
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 3648
                                }, this),
                                "8.3.2 / 2020-09-02"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 3623
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[35].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/react-component/dialog/pull/201",
                                        children: _changelogmdtypetext.texts[36].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 3808
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 3778
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 3774
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "831--2020-09-02",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#831--2020-09-02",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 3994
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 3934
                                }, this),
                                "8.3.1 / 2020-09-02"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 3909
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[37].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/react-component/dialog/pull/200",
                                        children: _changelogmdtypetext.texts[38].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 4094
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 4064
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 4060
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "830--2020-08-31",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#830--2020-08-31",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 4280
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 4220
                                }, this),
                                "8.3.0 / 2020-08-31"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 4195
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[39].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                        children: _changelogmdtypetext.texts[40].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 4380
                                    }, this),
                                    _changelogmdtypetext.texts[41].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/react-component/dialog/pull/195",
                                        children: _changelogmdtypetext.texts[42].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 4445
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 4350
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 4346
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "820--2020-08-27",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#820--2020-08-27",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 4631
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 4571
                                }, this),
                                "8.2.0 / 2020-08-27"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 4546
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[43].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                        children: _changelogmdtypetext.texts[44].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 4731
                                    }, this),
                                    _changelogmdtypetext.texts[45].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/react-component/dialog/pull/197",
                                        children: _changelogmdtypetext.texts[46].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 4796
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 4701
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 4697
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "811--2020-08-19",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#811--2020-08-19",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 4982
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 4922
                                }, this),
                                "8.1.1 / 2020-08-19"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 4897
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[47].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/react-component/dialog/pull/194",
                                        children: _changelogmdtypetext.texts[48].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 5082
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 5052
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 5048
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "810--2020-07-09",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#810--2020-07-09",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 5268
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 5208
                                }, this),
                                "8.1.0 / 2020-07-09"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 5183
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                    children: _changelogmdtypetext.texts[49].value
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 5338
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                    children: [
                                        _changelogmdtypetext.texts[50].value,
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                            children: _changelogmdtypetext.texts[51].value
                                        }, void 0, false, {
                                            fileName: "docs/changelog.md",
                                            lineNumber: 26,
                                            columnNumber: 5403
                                        }, this),
                                        _changelogmdtypetext.texts[52].value,
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                            children: _changelogmdtypetext.texts[53].value
                                        }, void 0, false, {
                                            fileName: "docs/changelog.md",
                                            lineNumber: 26,
                                            columnNumber: 5468
                                        }, this),
                                        _changelogmdtypetext.texts[54].value
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 5373
                                }, this)
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 5334
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "800--2020-05-29",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#800--2020-05-29",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 5628
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 5568
                                }, this),
                                "8.0.0 / 2020-05-29"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 5543
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[55].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                        children: _changelogmdtypetext.texts[56].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 5728
                                    }, this),
                                    _changelogmdtypetext.texts[57].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                        children: _changelogmdtypetext.texts[58].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 5793
                                    }, this),
                                    _changelogmdtypetext.texts[59].value
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 5698
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 5694
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "770--2020-05-05",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#770--2020-05-05",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 5953
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 5893
                                }, this),
                                "7.7.0 / 2020-05-05"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 5868
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[60].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                        children: _changelogmdtypetext.texts[61].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 6053
                                    }, this),
                                    _changelogmdtypetext.texts[62].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                        children: _changelogmdtypetext.texts[63].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 6118
                                    }, this),
                                    _changelogmdtypetext.texts[64].value
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 6023
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 6019
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "740--2019-05-10",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#740--2019-05-10",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 6278
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 6218
                                }, this),
                                "7.4.0 / 2019-05-10"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 6193
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: _changelogmdtypetext.texts[65].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 6348
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 6344
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "730--2018-12-06",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#730--2018-12-06",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 6473
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 6413
                                }, this),
                                "7.3.0 / 2018-12-06"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 6388
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[66].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                        children: _changelogmdtypetext.texts[67].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 6573
                                    }, this),
                                    _changelogmdtypetext.texts[68].value
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 6543
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 6539
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "720--2018-07-30",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#720--2018-07-30",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 6733
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 6673
                                }, this),
                                "7.2.0 / 2018-07-30"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 6648
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[69].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_dumi.Link, {
                                        to: "/*https://github.com/react-component/dialog/pull/89",
                                        children: _changelogmdtypetext.texts[70].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 6833
                                    }, this),
                                    _changelogmdtypetext.texts[71].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/HeskeyBaozi",
                                        children: _changelogmdtypetext.texts[72].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 6955
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 6803
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 6799
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "710--2017-12-28",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#710--2017-12-28",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 7121
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 7061
                                }, this),
                                "7.1.0 / 2017-12-28"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 7036
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[73].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/react-component/dialog/pull/72",
                                        children: _changelogmdtypetext.texts[74].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 7221
                                    }, this),
                                    _changelogmdtypetext.texts[75].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/Rohanhacker",
                                        children: _changelogmdtypetext.texts[76].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 7337
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 7191
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 7187
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "700--2017-11-02",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#700--2017-11-02",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 7503
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 7443
                                }, this),
                                "7.0.0 / 2017-11-02"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 7418
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                    children: [
                                        _changelogmdtypetext.texts[77].value,
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                            href: "https://github.com/react-component/m-dialog",
                                            children: _changelogmdtypetext.texts[78].value
                                        }, void 0, false, {
                                            fileName: "docs/changelog.md",
                                            lineNumber: 26,
                                            columnNumber: 7603
                                        }, this),
                                        _changelogmdtypetext.texts[79].value
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 7573
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                    children: _changelogmdtypetext.texts[80].value
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 7718
                                }, this)
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 7569
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: [
                                _changelogmdtypetext.texts[81].value,
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    href: "https://github.com/facebook/react/issues/10826",
                                    children: _changelogmdtypetext.texts[82].value
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 7787
                                }, this)
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 7758
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "6511--2017-8-21",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#6511--2017-8-21",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 7963
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 7903
                                }, this),
                                "6.5.11 / 2017-8-21"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 7878
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: [
                                    _changelogmdtypetext.texts[83].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/react-component/dialog/pull/64",
                                        children: _changelogmdtypetext.texts[84].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 8063
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 8033
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 8029
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "650--2016-10-25",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#650--2016-10-25",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 8248
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 8188
                                }, this),
                                "6.5.0 / 2016-10-25"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 8163
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: _changelogmdtypetext.texts[85].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 8318
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 8314
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "640--2016-09-19",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#640--2016-09-19",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 8443
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 8383
                                }, this),
                                "6.4.0 / 2016-09-19"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 8358
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: _changelogmdtypetext.texts[86].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 8513
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 8509
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "620--2016-07-18",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#620--2016-07-18",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 8638
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 8578
                                }, this),
                                "6.2.0 / 2016-07-18"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 8553
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: _changelogmdtypetext.texts[87].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 8708
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 8704
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "600--2016-03-18",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#600--2016-03-18",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 8833
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 8773
                                }, this),
                                "6.0.0 / 2016-03-18"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 8748
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                    children: _changelogmdtypetext.texts[88].value
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 8903
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                    children: _changelogmdtypetext.texts[89].value
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 8938
                                }, this)
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 8899
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "540--2016-02-27",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#540--2016-02-27",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 9063
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 9003
                                }, this),
                                "5.4.0 / 2016-02-27"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 8978
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: _changelogmdtypetext.texts[90].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 9133
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 9129
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "530--2015-11-23",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#530--2015-11-23",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 9258
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 9198
                                }, this),
                                "5.3.0 / 2015-11-23"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 9173
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: _changelogmdtypetext.texts[91].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 9328
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 9324
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "510--2015-10-20",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#510--2015-10-20",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 9453
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 9393
                                }, this),
                                "5.1.0 / 2015-10-20"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 9368
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: _changelogmdtypetext.texts[92].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 9523
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 9519
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "500--2015-08-17",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#500--2015-08-17",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 9648
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 9588
                                }, this),
                                "5.0.0 / 2015-08-17"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 9563
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                                children: _changelogmdtypetext.texts[93].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 26,
                                columnNumber: 9718
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 9714
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "450--2015-07-23",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#450--2015-07-23",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 9843
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 9783
                                }, this),
                                "4.5.0 / 2015-07-23"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 9758
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: _changelogmdtypetext.texts[94].value
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 9909
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "440--2015-07-03",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#440--2015-07-03",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 10027
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 9967
                                }, this),
                                "4.4.0 / 2015-07-03"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 9942
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: _changelogmdtypetext.texts[95].value
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 10093
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "420--2015-06-09",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#420--2015-06-09",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 10211
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 10151
                                }, this),
                                "4.2.0 / 2015-06-09"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 10126
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: _changelogmdtypetext.texts[96].value
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 10277
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "400--2015-04-28",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#400--2015-04-28",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 10395
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 10335
                                }, this),
                                "4.0.0 / 2015-04-28"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 10310
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: [
                                _changelogmdtypetext.texts[97].value,
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    href: "https://github.com/yiminghe/dom-align",
                                    children: _changelogmdtypetext.texts[98].value
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 10490
                                }, this),
                                _changelogmdtypetext.texts[99].value
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 10461
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "300--2015-03-17",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#300--2015-03-17",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 10683
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 10623
                                }, this),
                                "3.0.0 / 2015-03-17"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 10598
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: _changelogmdtypetext.texts[100].value
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 10749
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "210--2015-03-05",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#210--2015-03-05",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 26,
                                        columnNumber: 10868
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 10808
                                }, this),
                                "2.1.0 / 2015-03-05"
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 10783
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                    children: _changelogmdtypetext.texts[101].value
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 10937
                                }, this),
                                _changelogmdtypetext.texts[102].value,
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    href: "https://github.com/react-component/dialog/issues/3",
                                    children: _changelogmdtypetext.texts[103].value
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 26,
                                    columnNumber: 11004
                                }, this),
                                _changelogmdtypetext.texts[104].value
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 26,
                            columnNumber: 10934
                        }, this)
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 26,
                    columnNumber: 11
                }, this)
            }, void 0, false)
        }, void 0, false, {
            fileName: "docs/changelog.md",
            lineNumber: 25,
            columnNumber: 7
        }, this)
    }, void 0, false, {
        fileName: "docs/changelog.md",
        lineNumber: 24,
        columnNumber: 5
    }, this);
}
_c = DumiMarkdownContent;
var _default = DumiMarkdownContent;
var _c;
$RefreshReg$(_c, "DumiMarkdownContent");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
 }]);
