// @rc-component/dialog/dist/docs_index_md-async.js
((typeof globalThis !== 'undefined' ? globalThis : self)["makoChunk_@rc-component/dialog"] = (typeof globalThis !== 'undefined' ? globalThis : self)["makoChunk_@rc-component/dialog"] || []).push([
        ['docs/index.md'],
{ "README.md?watch=parent": function (module, exports, __mako_require__){

},
"docs/index.md": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/react/jsx-dev-runtime.js");
var _index = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/dumi/theme-default/builtins/API/index.js"));
var _index1 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/dumi/theme-default/builtins/Badge/index.js"));
var _index2 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/dumi/theme-default/builtins/CodeGroup/index.js"));
var _index3 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/dumi/theme-default/builtins/Container/index.js"));
var _index4 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/dumi/theme-default/builtins/Previewer/index.js"));
var _index5 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/dumi/theme-default/builtins/SourceCode/index.js"));
var _index6 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/dumi/theme-default/builtins/Table/index.js"));
var _index7 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/dumi/theme-default/builtins/Tree/index.js"));
var _dumi = __mako_require__(".dumi/tmp/dumi/exports.ts");
__mako_require__("README.md?watch=parent");
var _loading = /*#__PURE__*/ _interop_require_default._(__mako_require__(".dumi/tmp/dumi/theme/loading.tsx"));
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react/index.js"));
var _indexmdtypetext = __mako_require__("docs/index.md?type=text");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
// export named function for fastRefresh
// ref: https://github.com/pmmmwh/react-refresh-webpack-plugin/blob/main/docs/TROUBLESHOOTING.md#edits-always-lead-to-full-reload
function DumiMarkdownContent() {
    return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_dumi.DumiPage, {
        children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_react.Suspense, {
            fallback: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_loading.default, {}, void 0, false, {
                fileName: "docs/index.md",
                lineNumber: 25,
                columnNumber: 27
            }, void 0),
            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_jsxdevruntime.Fragment, {
                children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
                    className: "markdown",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h1", {
                            id: "rc-componentdialog",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#rc-componentdialog",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 26,
                                        columnNumber: 128
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 65
                                }, this),
                                "@rc-component/dialog"
                            ]
                        }, void 0, true, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 37
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: _indexmdtypetext.texts[0].value
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 196
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    href: "https://npmjs.org/package/@rc-component/dialog",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("img", {
                                        src: "https://img.shields.io/npm/v/@rc-component/dialog.svg?style=flat-square",
                                        alt: "NPM version"
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 26,
                                        columnNumber: 288
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 231
                                }, this),
                                _indexmdtypetext.texts[1].value,
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    href: "https://npmjs.org/package/@rc-component/dialog",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("img", {
                                        src: "https://img.shields.io/npm/dm/@rc-component/dialog.svg?style=flat-square",
                                        alt: "npm download"
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 26,
                                        columnNumber: 477
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 420
                                }, this),
                                _indexmdtypetext.texts[2].value,
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    href: "https://github.com/react-component/dialog/actions/workflows/react-component-ci.yml",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("img", {
                                        src: "https://github.com/react-component/dialog/actions/workflows/react-component-ci.yml/badge.svg",
                                        alt: "build status"
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 26,
                                        columnNumber: 704
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 611
                                }, this),
                                _indexmdtypetext.texts[3].value,
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    href: "https://app.codecov.io/gh/react-component/dialog",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("img", {
                                        src: "https://img.shields.io/codecov/c/github/react-component/dialog/master.svg?style=flat-square",
                                        alt: "Codecov"
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 26,
                                        columnNumber: 917
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 858
                                }, this),
                                _indexmdtypetext.texts[4].value,
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    href: "https://bundlephobia.com/package/@rc-component/dialog",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("img", {
                                        src: "https://badgen.net/bundlephobia/minzip/@rc-component/dialog",
                                        alt: "bundle size"
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 26,
                                        columnNumber: 1129
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 1065
                                }, this),
                                _indexmdtypetext.texts[5].value,
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    href: "https://github.com/umijs/dumi",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("img", {
                                        src: "https://img.shields.io/badge/docs%20by-dumi-blue?style=flat-square",
                                        alt: "dumi"
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 26,
                                        columnNumber: 1289
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 1249
                                }, this)
                            ]
                        }, void 0, true, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 228
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "screenshot",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#screenshot",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 26,
                                        columnNumber: 1463
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 1408
                                }, this),
                                "Screenshot"
                            ]
                        }, void 0, true, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 1388
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("img", {
                            src: "http://gtms04.alicdn.com/tps/i4/TB1dp5lHXXXXXbmXpXXyVug.FXX-664-480.png"
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 1521
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "example",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#example",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 26,
                                        columnNumber: 1675
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 1623
                                }, this),
                                "Example"
                            ]
                        }, void 0, true, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 1606
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "http://localhost:8007/examples/",
                                children: _indexmdtypetext.texts[6].value
                            }, void 0, false, {
                                fileName: "docs/index.md",
                                lineNumber: 26,
                                columnNumber: 1733
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 1730
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: [
                                _indexmdtypetext.texts[7].value,
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    href: "https://dialog.react-component.vercel.app/",
                                    children: _indexmdtypetext.texts[8].value
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 1836
                                }, this)
                            ]
                        }, void 0, true, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 1808
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "install",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#install",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 26,
                                        columnNumber: 1991
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 1939
                                }, this),
                                "Install"
                            ]
                        }, void 0, true, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 1922
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://npmjs.org/package/@rc-component/dialog",
                                children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("img", {
                                    src: "https://nodei.co/npm/@rc-component/dialog.png",
                                    alt: "@rc-component/dialog"
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 2106
                                }, this)
                            }, void 0, false, {
                                fileName: "docs/index.md",
                                lineNumber: 26,
                                columnNumber: 2049
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 2046
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "usage",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#usage",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 26,
                                        columnNumber: 2265
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 2215
                                }, this),
                                "Usage"
                            ]
                        }, void 0, true, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 2200
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_index5.default, {
                            lang: "js",
                            children: _indexmdtypetext.texts[9].value
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 2318
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "api",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#api",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 26,
                                        columnNumber: 2439
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 2391
                                }, this),
                                "API"
                            ]
                        }, void 0, true, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 2378
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h3", {
                            id: "rc-componentdialog-1",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#rc-componentdialog-1",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 26,
                                        columnNumber: 2585
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 2520
                                }, this),
                                "@rc-component/dialog"
                            ]
                        }, void 0, true, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 2490
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_index6.default, {
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("thead", {
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                        children: [
                                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("th", {
                                                children: _indexmdtypetext.texts[10].value
                                            }, void 0, false, {
                                                fileName: "docs/index.md",
                                                lineNumber: 26,
                                                columnNumber: 2671
                                            }, this),
                                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("th", {
                                                children: _indexmdtypetext.texts[11].value
                                            }, void 0, false, {
                                                fileName: "docs/index.md",
                                                lineNumber: 26,
                                                columnNumber: 2706
                                            }, this),
                                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("th", {
                                                children: _indexmdtypetext.texts[12].value
                                            }, void 0, false, {
                                                fileName: "docs/index.md",
                                                lineNumber: 26,
                                                columnNumber: 2741
                                            }, this),
                                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("th", {
                                                children: _indexmdtypetext.texts[13].value
                                            }, void 0, false, {
                                                fileName: "docs/index.md",
                                                lineNumber: 26,
                                                columnNumber: 2776
                                            }, this),
                                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("th", {
                                                children: _indexmdtypetext.texts[14].value
                                            }, void 0, false, {
                                                fileName: "docs/index.md",
                                                lineNumber: 26,
                                                columnNumber: 2811
                                            }, this)
                                        ]
                                    }, void 0, true, {
                                        fileName: "docs/index.md",
                                        lineNumber: 26,
                                        columnNumber: 2667
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 2660
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tbody", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[15].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 2870
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[16].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 2905
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[17].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 2940
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[18].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 2975
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3010
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 2866
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[19].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3025
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[20].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3060
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3095
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[21].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3101
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3136
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 3021
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[22].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3151
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[23].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3186
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3221
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[24].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3227
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3262
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 3147
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[25].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3277
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[26].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3312
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3347
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[27].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3353
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3388
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 3273
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[28].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3403
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[29].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3438
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[30].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3473
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[31].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3508
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3543
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 3399
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[32].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3558
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[33].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3593
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3628
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3634
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3640
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 3554
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[34].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3655
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[35].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3690
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[36].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3725
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[37].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3760
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3795
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 3651
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[38].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3810
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[39].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3845
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3880
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[40].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3886
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3921
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 3806
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[41].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3936
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[42].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 3971
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4006
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[43].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4012
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4047
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 3932
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[44].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4062
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[45].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4097
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4132
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[46].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4138
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4173
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 4058
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[47].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4188
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[48].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4223
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4258
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[49].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4264
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4299
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 4184
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[50].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4314
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[51].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4349
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4384
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[52].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4390
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4425
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 4310
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[53].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4440
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[54].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4475
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4510
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[55].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4516
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4551
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 4436
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[56].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4566
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[57].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4601
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[58].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4636
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[59].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4671
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4706
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 4562
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[60].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4721
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[61].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4756
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[62].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4791
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[63].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4826
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4861
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 4717
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[64].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4876
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[65].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4911
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[66].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4946
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[67].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 4981
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5016
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 4872
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[68].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5031
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[69].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5066
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[70].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5101
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[71].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5136
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5171
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 5027
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[72].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5186
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[73].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5221
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5256
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[74].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5262
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5297
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 5182
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[75].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5312
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[76].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5347
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5382
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[77].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5388
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5423
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 5308
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[78].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5438
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[79].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5473
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5508
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[80].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5514
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5549
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 5434
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[81].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5564
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[82].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5599
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5634
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[83].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5640
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5675
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 5560
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[84].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5690
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[85].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5725
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[86].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5760
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[87].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5795
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5830
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 5686
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[88].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5845
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[89].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5880
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5915
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[90].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5921
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5956
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 5841
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[91].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 5971
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[92].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 6006
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[93].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 6041
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[94].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 6076
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 6111
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 5967
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[95].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 6126
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[96].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 6161
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[97].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 6196
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[98].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 6231
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 6266
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 6122
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                            children: [
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[99].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 6281
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[100].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 6316
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 6352
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[101].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 6358
                                                }, this),
                                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                                    children: _indexmdtypetext.texts[102].value
                                                }, void 0, false, {
                                                    fileName: "docs/index.md",
                                                    lineNumber: 26,
                                                    columnNumber: 6394
                                                }, this)
                                            ]
                                        }, void 0, true, {
                                            fileName: "docs/index.md",
                                            lineNumber: 26,
                                            columnNumber: 6277
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 2859
                                }, this)
                            ]
                        }, void 0, true, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 2653
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "development",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#development",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 26,
                                        columnNumber: 6528
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 6472
                                }, this),
                                "Development"
                            ]
                        }, void 0, true, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 6451
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_index5.default, {
                            children: _indexmdtypetext.texts[103].value
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 6587
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "test-case",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#test-case",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 26,
                                        columnNumber: 6712
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 6658
                                }, this),
                                "Test Case"
                            ]
                        }, void 0, true, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 6639
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_index5.default, {
                            children: _indexmdtypetext.texts[104].value
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 6769
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "coverage",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#coverage",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 26,
                                        columnNumber: 6892
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 6839
                                }, this),
                                "Coverage"
                            ]
                        }, void 0, true, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 6821
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_index5.default, {
                            children: _indexmdtypetext.texts[105].value
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 6948
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: _indexmdtypetext.texts[106].value
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 7000
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "license",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#license",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 26,
                                        columnNumber: 7103
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 7051
                                }, this),
                                "License"
                            ]
                        }, void 0, true, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 7034
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                            children: _indexmdtypetext.texts[107].value
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 7158
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                            id: "-contributing",
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    "aria-hidden": "true",
                                    tabIndex: "-1",
                                    href: "#-contributing",
                                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                        className: "icon icon-link"
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 26,
                                        columnNumber: 7273
                                    }, this)
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 26,
                                    columnNumber: 7215
                                }, this),
                                " Contributing"
                            ]
                        }, void 0, true, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 7192
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            href: "https://openomy.app/github/react-component/dialog",
                            target: "_blank",
                            style: {
                                display: "block",
                                width: "100%"
                            },
                            align: "center",
                            children: [
                                _indexmdtypetext.texts[108].value,
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("img", {
                                    src: "https://www.openomy.app/svg?repo=react-component/dialog&chart=bubble&latestMonth=24",
                                    target: "_blank",
                                    alt: "Contribution Leaderboard",
                                    style: {
                                        display: "block",
                                        width: "100%"
                                    }
                                }, void 0, false, {
                                    fileName: "docs/index.md",
                                    lineNumber: 29,
                                    columnNumber: 46
                                }, this)
                            ]
                        }, void 0, true, {
                            fileName: "docs/index.md",
                            lineNumber: 26,
                            columnNumber: 7336
                        }, this)
                    ]
                }, void 0, true, {
                    fileName: "docs/index.md",
                    lineNumber: 26,
                    columnNumber: 11
                }, this)
            }, void 0, false)
        }, void 0, false, {
            fileName: "docs/index.md",
            lineNumber: 25,
            columnNumber: 7
        }, this)
    }, void 0, false, {
        fileName: "docs/index.md",
        lineNumber: 24,
        columnNumber: 5
    }, this);
}
_c = DumiMarkdownContent;
var _default = DumiMarkdownContent;
var _c;
$RefreshReg$(_c, "DumiMarkdownContent");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
 }]);
