// @rc-component/dialog/dist/dumi__tmp__dumi__theme__ContextWrapper-async.js
((typeof globalThis !== 'undefined' ? globalThis : self)["makoChunk_@rc-component/dialog"] = (typeof globalThis !== 'undefined' ? globalThis : self)["makoChunk_@rc-component/dialog"] || []).push([
        ['dumi__tmp__dumi__theme__ContextWrapper'],
{ ".dumi/tmp/dumi/meta/atoms.ts": function (module, exports, __mako_require__){
// @ts-nocheck
// This file is generated by Umi automatically
// DO NOT CHANGE IT MANUALLY!
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "components", {
    enumerable: true,
    get: function() {
        return components;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const components = null;
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
".dumi/tmp/dumi/theme/ContextWrapper.tsx": function (module, exports, __mako_require__){
// @ts-nocheck
// This file is generated by Umi automatically
// DO NOT CHANGE IT MANUALLY!
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return DumiContextWrapper;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/react/jsx-dev-runtime.js");
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/react/index.js"));
var _dumi = __mako_require__(".dumi/tmp/dumi/exports.ts");
var _rcutil = __mako_require__("node_modules/rc-util/es/index.js");
var _context = __mako_require__("node_modules/dumi/dist/client/theme-api/context.js");
var _atoms = __mako_require__(".dumi/tmp/dumi/meta/atoms.ts");
var _config = __mako_require__(".dumi/tmp/dumi/locales/config.ts");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _s = $RefreshSig$();
const entryExports = {};
// Static content
const pkg = {
    "name": "@rc-component/dialog",
    "description": "dialog ui component for react",
    "version": "1.5.1",
    "license": "MIT",
    "repository": {
        "type": "git",
        "url": "git@github.com:react-component/dialog.git"
    },
    "author": "yiminghe@gmail.com"
};
const historyType = "browser";
const hostname = undefined;
const themeConfig = {
    "footer": "Copyright © 2025 | Powered by <a href=\"https://d.umijs.org\" target=\"_blank\" rel=\"noreferrer\">dumi</a>",
    "prefersColor": {
        "default": "light",
        "switch": true
    },
    "nprogress": true,
    "lastUpdated": true,
    "name": "Dialog",
    "logo": "https://avatars0.githubusercontent.com/u/9441414?s=200&v=4",
    "sourceLink": "https://github.com/react-component/dialog/tree/focus/{fileName}#L{line}",
    "editLink": "https://github.com/react-component/dialog/edit/focus/{filename}"
};
const _2_level_nav_available = false;
function DumiContextWrapper() {
    _s();
    const outlet = (0, _dumi.useOutlet)();
    const [loading, setLoading] = (0, _react.useState)(false);
    const prev = (0, _react.useRef)(_dumi.history.location.pathname);
    (0, _react.useEffect)(()=>{
        return _dumi.history.listen((next)=>{
            if (next.location.pathname !== prev.current) {
                prev.current = next.location.pathname;
                // scroll to top when route changed
                document.documentElement.scrollTo(0, 0);
            }
        });
    }, []);
    const context = _react.default.useMemo(()=>{
        const ctx = {
            pkg,
            historyType,
            entryExports,
            demos: null,
            components: _atoms.components,
            locales: _config.locales,
            loading,
            setLoading,
            hostname,
            themeConfig,
            _2_level_nav_available
        };
        // Proxy do not warning since `Object.keys` will get nothing to loop
        Object.defineProperty(ctx, 'demos', {
            get: ()=>{
                (0, _rcutil.warning)(false, '`demos` return empty in latest version, please use `useDemo` instead.');
                return {};
            }
        });
        return ctx;
    }, [
        pkg,
        historyType,
        entryExports,
        _atoms.components,
        _config.locales,
        loading,
        setLoading,
        hostname,
        themeConfig,
        _2_level_nav_available
    ]);
    return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_context.SiteContext.Provider, {
        value: context,
        children: outlet
    }, void 0, false, {
        fileName: ".dumi/tmp/dumi/theme/ContextWrapper.tsx",
        lineNumber: 77,
        columnNumber: 5
    }, this);
}
_s(DumiContextWrapper, "hiffCkZ+p3FfU1Rtsjsx45houdE=", false, function() {
    return [
        _dumi.useOutlet
    ];
});
_c = DumiContextWrapper;
var _c;
$RefreshReg$(_c, "DumiContextWrapper");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
 }]);
